streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -1088,7 +1107,6 @@ public CustomProcessorInfo removeCustomProcessorInfoAsBundle(String name) throws
             }
         }
         CustomProcessorInfo customProcessorInfo = new CustomProcessorInfo();
-        deleteFileFromStorage(customProcessorInfo.fromTopologyComponentBundle(customProcessorBundle).getJarFileName());
         this.removeTopologyComponentBundle(customProcessorBundle.getId());
         return customProcessorInfo;
```

Comment:
Empty `customProcessorInfo` is getting returned from here with these changes. We should not allow default no args constructor for `CustomProcessorInfo` to be accessible. It is good to have factory method to create instance from TopologyComponentBundle as I mentioned in other comment in `CustomProcessorInfo`

Reply:
I had removed it since i did not think it was needed anymore as jar file name is not needed for deletion anymore. For now, I have changed it back to not return an empty CustomProcessorInfo